File: CPlusPlus/Junior/2026/OlympicScores.cpp
/**
 *      CCC 2026 Junior - Olympic Scores
 *
 */

#include <bits/stdc++.h>

using namespace std;

int main()
{
    int scores[5];
    int D;
    for (int i = 0; i < 5; i++)
    {
        cin >> scores[i];
    }
    cin >> D;

    sort(scores, scores + 5);

    int result = 0;
    for (int i = 1; i < 4; i++)
    {
        result += scores[i];
    }
    result *= D;

    cout << result << endl;

    return 0;
}

File: Java/Junior/2026/OlympicScores.java
/**
 *      CCC 2026 Junior - Olympic Scores
 *
 */

import java.io.*;
import java.lang.*;
import java.util.*;

public class Main {

    public static void main (String [] args) throws Exception {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int[] scores = new int[5];
        for (int i = 0; i < 5; i++) 
        {
            scores[i] = Integer.parseInt(br.readLine());
        }
        
        int D = Integer.parseInt(br.readLine());

        Arrays.sort(scores);

        int result = 0;
        for (int i = 1; i <= 3; i++) 
        {
            result += scores[i];
        }

        result *= D;

        System.out.println(result);   
    }
}